Add xyz.fontra.FontraPak

mitradranirban opened the pull request:
<!-- ⚠️⚠️  Submission pull request MUST be made against the `new-pr` **base branch** ⚠️⚠️  -->

<!-- 💡 Please go to the preview tab to view the markdown below 💡 -->

### Please confirm your submission meets all the criteria

<!-- 💡 Please replace each `[ ]` with `[X]` when the step is complete 💡 -->

<!-- 💡 Please tick and write 'N/A' with a reason if a checklist item below is not applicable 💡 -->

- [X] Please describe the application briefly. Fontra Pak is a cross-platform, standalone, bundled Font Editor application for desktop use.
- [X] Please attach a video showcasing the application on Linux using the Flatpak. 

https://github.com/user-attachments/assets/dfa0d80f-a659-46fa-88f8-25fe093ef01e


- [X] The Flatpak ID follows all the rules listed in the [Application ID requirements][appid].
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2] and I agree to them.
- [X] I contacted upstream developers about this submission. [**Link:**](https://github.com/fontra/fontra-pak/issues/193)

<!-- 💡 Please mention below the GitHub usernames of any additional maintainers needed (if any) 💡 -->

<!-- ⚠️⚠️  Please DO NOT modify anything below this line ⚠️⚠️  -->

[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission

hfiguiere commented on data/xyz.fontra.FontraPak.desktop:
this file is duplicate. and never used.

hfiguiere commented on data/icons/fontrapak.svg:
This file should be (submitted and managed) upstream.

hfiguiere replied:
how is an SVG file linux specific? Where does it originate?

mitradranirban commented:
Modifications done 
1. updated runtime to 25.08
2. added separate script file and removed  creating script 
3. removed windows and MacOS icon file and extra copies of desktop file 
4. Changed licence 
`build, bot`

mitradranirban commented:
`bot, build` - Restart the test build

mitradranirban commented:
bot, build

mitradranirban commented:
bot, build

mitradranirban commented:
Thoroughly bug fixed and tested Fit for review

hfiguiere commented:
still many unanswered questions.

mitradranirban commented on xyz.fontra.FontraPak.appdata.xml:
```diff
@@ -0,0 +1,29 @@
+  </releases>
+  <content_rating type="oars-1.1"/>
+  <translation type="gettext"/>
+  <project_license>CC-BY-4.0</project_license>
```
Corrected metadata and project licence. Metainfo file validates

hfiguiere commented on xyz.fontra.FontraPak.metainfo.xml:
```diff
@@ -0,0 +1,37 @@
+  <content_rating type="oars-1.1"/>
+  <screenshots>
+   <screenshot type="default">
+   <image>https://raw.githubusercontent.com/mitradranirban/flathub/refs/heads/my-app-submission/screenshot1.png</image>
```
URL should NEVER reference the tip of a branch. It's not immutable.

hfiguiere commented on xyz.fontra.FontraPak.yml:
```diff
@@ -0,0 +1,50 @@
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/mitradranirban/fontra-pak-linux/releases/download/2025.11.0/fontrapak.tgz
```
also why is this not from upstream repository ?

mitradranirban commented on xyz.fontra.FontraPak.metainfo.xml:
Screenshot added, now validating ok

mitradranirban commented:
As this is not acceptable to Flathub I am closing this pull request
